### PR TITLE
use hooks

### PR DESCRIPTION
**Pull request by Kamahl19**
#101

### CODE REVIEW

File: src/packages/spinner/ducks.js [hunk 1]
```diff
@@ -57,39 +55,5 @@ export default combineReducers({
  * SELECTORS
  */
 const selectSpinner = state => state.spinner;
-const selectIsInProgress = (state, apiCallId) => !!selectSpinner(state).apiCalls[apiCallId];
-const selectGlobalCounter = state => !!selectSpinner(state).globalCounter;
-
-/**
- * DECORATORS
- */
-export function withApiCall(apiCallId, saga) {
```
Comment (MiroslavPetrik) [inline src/packages/spinner/ducks.js]: why removing this?

Comment (Kamahl19) [reply src/packages/spinner/ducks.js]: Its not uses anywhere and i dont want to tight spinner pkg to saga. Anyone can create this helper in app code

File: src/app/Root.js [hunk 2]
```diff
@@ -54,3 +50,13 @@ const Root = () => (
 );
 
 export default Root;
+
+const GlobalSpinnerProvider = ({ children }) => {
+  const isVisible = useSpinner();
```
[the remark this answers is not in the document]
Comment (Kamahl19) [reply src/app/Root.js]: Good catch

[the remark this answers is not in the document]
Comment (Kamahl19) [reply src/app/Root.js]: Good catch

**Pull request merged**